server/reports/README.md

```diff
@@ -0,0 +1,451 @@
+
+Open mSupply standard reports are embedded during building from the `reports/generated/standard-reports.json` file. Embedded reports are included in release builds, and are therefore accessible on testing builds where the OMS repo file structure is not available.
+
+This command is used to update standard reports to the current branch in databases on live or test builds.
+
+### Show Report
+
+`show-report --path <path-to-report-dir-containing-report-manifest.json> --config <optional-path-to-dir-containing-test-config.json>
+
+Show report replaces previously used print.sh and show.sh bash commands on the OMS reports repo.
+
+Running this command will generate and open an html file of the report.
+
+This command uses a `test-config.json` file located in the reports dir containing all arguments used to generate the report.
+
+A custom test-config.json file can be used to render with specific arguments by passing a path to a dir containing a report specific `test-config.json file`
+
+## Development
+
+Other functionality, and processes used in report development are:
+
+[Translating of reports](#translating-reports)
+[Standard vs custom reports]
+[Report versioning](#report-versioning)
+[Wasm functions](#wasm-functions)
+[Development processes](#development-processes)
+[File Structure]
+
+### Translating reports
+
+Reports have the option to allow for translations using the same localisation suite we use for front end translations.
+
+Translating functionality should be used in standard reports. Custom reports for specific clients typically are hard coded.
+
+##### Translating function
+
+This can be implemented in the report by adding the following translation function in place of your text:
+
+```
+{{t(k="label.name", f="Name")}}
+```
+
+Where the letters are short hand for the following:
+
+- t for translate
+  The name of the function
+- k for key
+  This is the locale key as is used in front end translations.
+- f for fallback
+  This is an optional fallback text if the translation cannot be found.
+- n for namespace (depreciated - only common.json is used)
+  The file namespace where the translation key is. The .json extension is automatically added i.e. catalogue (which refers to the catalogue.json namespace).
+  By default, the translation in common.json translations will be used.
+  Note all translation files have been consolidated into the common.json namespace, so the namespace parameter is no longer used. However, functionality is retained in case we are needing customer specific namespaces in future.
+
+The current user language is passed through GraphQL when a user requests a report to be generated. This is the language used in translations.
+
+##### Translating search pattern
+
+The translation function has a number of fallback translations which it will search through if the translation cannot be found.
+
+First it will look for the translation key within the nominated namespace and language
```

Comment:
Need to add comment about dialect, Lache added this in recently to load fr-DJ translations (: